buddyboss/buddyboss-platform
# Member search for learndash open course

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-app/issues/1919

### How to test the changes in this Pull Request:

When any learndash course access mode is set as 'open' then its enrolled user does not appear in members search result. 

`The Course price type is set to "open". This means ALL are automatically enrolled`

https://www.loom.com/share/ff93e1ed0d854d1dbd58bc32854d4d3b

### Proof Screenshots or Video

https://www.loom.com/share/1353810069c646be8982dcf083ef5a5b

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.


### Related Platform API PR

https://github.com/buddyboss/buddyboss-platform-api/pull/246
